content/development/handbook/back-end/error-handling/_index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Error handling and Error logging
+description: Guidelines for error handling and error logging in backend
+tags: ["development", "handbook", "back-end", "error-handling", "error-logging"]
+weight: 100
+---
+{{%notice warning%}}
+This page is a work-in-progress. Because we do not have details on how potential errors should be shown to the users, this is only a general overview currently. More details will be added once we have a functional understanding of how users should experience errors. 
+{{% /notice%}}
+
+#### Exception Handling
+.Net core provides a default middleware "UseExceptionHandler" that will catch exceptions and log them. You can read more about it [here](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/error-handling?view=aspnetcore-2.2).
+In addition to the default exception middleware, a custom exception middleware can be added to customize the error handling to suit application needs. Customm exception middleware will be implemented once we have a functional understanding of how users should experience errors.
+
+### API
+Exception in API is handled by sending response message to users. The response messages are sent based on 
+
+- Request validation 
+- Input validation
+- Exceptions that is thrown due to bug in code
+
+The repsonse messages usually are returned with a status code. It is recommended to start with the basic status codes below
+
+- 200 - Ok (Success)
+- 400 - Bad request (input validation, request validation (basically problems at client side))
+- 401 - Unauthorized
+- 404 - Not found
+- 403 - Forbidden
```

Comment:
Move it over `- 404`? Since the number 403 is lower than 404.

Reply:
Fixed